### DISCUSSION TITLE
No response to github ticket

### DISCUSSION THREAD

**Discussion by alexanderharringII** [community/community#137608] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

My ticket has not got a response for months and I have bumped it multiple times.

Has anyone else experienced this and how do I get support?

**Comment by ThisIs-Developer** [comment]
Hello, I got the same problem a little while ago. What I did is my account was suspended in that I could not Log In as well. By far the simplest and most straightforward case to open a ticket was on the GitHub (compile), I did not need to log in. But alas, I didn’t receive any response from GitHub for quite some time. The last one is even I contacted some people on social media some who are fake claiming they can help me by charging me some amount of money. 
 
 This remained the experience for about three months until GitHub’s support team wrote back. They simply had to confirm my e–mail address , and my account was reinstated. 
 
 Therefore, in case of GitHub, it is wise that you wait for the support to respond to your issue. You can’t really do more than this, and it is advisable to stay away from any third party tricks that claim to have a solution for the problem for a fee. 
 
 Stay as strong as you are and hope your problem gets solved as early as possible!

**Reply by alexanderharringII** [reply]
The ticket was for getting my account back after someone hacked it, changed the emails and got it suspended. My ticket has been unanswered for over a month now, so not quite your three months just yet. Hopefully they will respond soon because waiting 3 months for a response is shocking

**Comment by ebndev** [comment]
Hi @alexanderharringII,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
